packages/wallet-backend-update/tests/auth/controller.test.ts

```diff
@@ -0,0 +1,192 @@
+import { createContainer } from '@/createContainer'
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { Container } from '@/shared/container'
+import { createApp, TestApp } from '../app'
+import { Knex } from 'knex'
+import { truncateTables } from '../tables'
+import { Request, Response } from 'express'
+import {
+  createRequest,
+  createResponse,
+  MockRequest,
+  MockResponse
+} from 'node-mocks-http'
+import type { AuthController } from '@/auth/controller'
+import type { AuthService } from '@/auth/service'
+import { errorHandler } from '@/middleware/errorHandler'
+import { faker } from '@faker-js/faker'
+import { applyMiddleware } from '../utils'
+import { withSession } from '@/middleware/withSession'
+import type { UserService } from '@/user/service'
+
+describe('Authentication Controller', (): void => {
+  let bindings: Container<Bindings>
+  let appContainer: TestApp
+  let knex: Knex
+  let authService: AuthService
+  let authController: AuthController
+  let userService: UserService
+  let req: MockRequest<Request>
+  let res: MockResponse<Response>
+
+  const next = jest.fn()
+  const args = {
+    email: faker.internet.email(),
+    password: faker.internet.password()
+  }
```

Comment:
Could we create a mock file, and put args in it? So that it can be imported in several test files..